libraries/Ethernet/src/EthernetClock.h

```diff
@@ -0,0 +1,25 @@
+
+#ifndef ETHERNET_CLOCK_H
+#define ETHERNET_CLOCK_H
+
+
+#include "FspTimer.h"
+
+#define AGT_TIMER_CHANNEL 3
+#define ETHERNET_CLK_PIN  BSP_IO_PORT_06_PIN_00
```

Comment:
I'd move these defines in the variant folder (in `pins_arduino.h`), since the library might also run on HW that doesn't use the same pins.